### PR TITLE
stone-soup 0.16.2

### PR DESCRIPTION
**Pull request by tomyun**
#326 replayed:

- Use brewed or OS X native deps instead of internal builds
- `with-tiles` added to support graphics mode
- Sound support is implied with tiles (no way to enable only sound, no graphics)
- `DATADIR` changed from `data` to `share` (following brewed `pkgshare`)
- Remove no longer needed SDK path
- `ENV.libstdcxx` changed to `ENV.cxx11` (only Mountain Lion needs it)
- `test` added to check version

### CODE REVIEW

**Comment by tomyun** [comment]
Build bots failed with Jenkins error and not recovered by pushing another commits. Can someone handle this issue?

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

File: stone-soup.rb [hunk 1]
```diff
@@ -11,25 +10,49 @@ class StoneSoup < Formula
...
 
     cd "source" do
-      # The makefile has trouble locating the developer tools for
-      # CLT-only systems, so we set these manually. Reported upstream:
-      # https://crawl.develz.org/mantis/view.php?id=7625
-      #
-      # On 10.9, stone-soup will try to use xcrun and fail due to an empty
-      # DEVELOPER_DIR
-      devdir = MacOS::Xcode.prefix.to_s
-      devdir += '/' if MacOS.version >= :mavericks && !MacOS::Xcode.installed?
```
Comment (mistydemeo) [inline stone-soup.rb]: Do you mind leaving this part in? The new version still fails to build on a CLT-only system for me.

Comment (tomyun) [inline stone-soup.rb]: Didn't know this issue on a CLT-only system. The formula is now updated to keep this part in. Could you restart the build bots? They failed again with a forced push commit.

**Comment by mistydemeo** [comment]
Thanks!

**Pull request closed without merging**